### ISSUE TITLE
`dap-variables-expand-in-launch-configuration` appends nil when `:command` is a list of a single element

### ISSUE THREAD

**Issue by danielmartin** [emacs-lsp/dap-mode#346]
Steps to reproduce the problem:

Just eval this from the scratch buffer:

```
(dap-variables-expand-in-launch-configuration '(:name "Debug"
 :connection-type 'pipe
 :coding 'no-conversion
 :command '("lldb-vscode")
 :stderr "*Whatever stderr*"
 :noquery t))
```

Note that the resulting form has the following `:command`:

```
[...]
:command '("lldb-vscode" nil)
[...]
```

This nil element causes an error later on when a call to `make-process` is built and run.

**Comment by yyoncho** [comment]
cc @nbfalcon

**Comment by nbfalcon** [comment]
Fixed in #351.

Sorry for being so late to reply; I am currently on holiday, which means I don't have too much time to hack on dap-mode. This time I added some tests, and @danielmartin's case is handled correctly now.
